controllers/scripts/create_pod_status_patch.py

```
@@ -446,60 +472,76 @@ def wipe_volumes(pod_name, config):
+                                      f"- Mounting point does not exists")
+                        raise FileNotFoundError(f"{volume} Volume path not found")
+
+                    for ns_file_path in filter(lambda x: x.startswith(volume.get_attachment_path()), ns_file_paths):
+                        _, filename = os.path.split(ns_file_path)
+                        file_path = os.path.join(volume.get_mount_point(), filename)
+                        if os.path.exists(file_path):
+                            logging.info(f"Submitting file: - {file_path}")
+                            futures[executor.submit(lambda: os.remove(file_path))] = file_path
+                        else:
+                            logging.warning(f"{volume} namespace-file-path: {file_path} - Does not exists")
 
+                else:
+                    logging.error(f"{volume} - Has invalid effective method")
+                    raise ValueError(f"{volume} - Has invalid effective method")
             else:
-                logging.error(f"{volume} - Has invalid effective method")
-                raise ValueError(f"{volume} - Has invalid effective method")
-        else:
-            logging.error(f"pod-name: {pod_name} Invalid volume-mode: {volume.volume_mode}")
-            raise ValueError(f"pod-name: {pod_name} Invalid volume-mode: {volume.volume_mode}")
+                logging.error(f"pod-name: {pod_name} Invalid volume-mode: {volume.volume_mode}")
+                raise ValueError(f"pod-name: {pod_name} Invalid volume-mode: {volume.volume_mode}")
+
+        for future in concurrent.futures.as_completed(fs=futures):
+            cmd = futures[future]
+            try:
+                if future.done():
+                    logging.info(f"pod-name: {pod_name} Finished Successfully: {cmd}")
+            except Exception as e:
+                logging.error(f"pod-name: {pod_name} Error running: {cmd} Error: {e}")
```

Should we raise an error here?